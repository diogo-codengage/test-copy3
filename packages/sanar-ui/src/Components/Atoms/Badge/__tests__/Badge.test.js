import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESBadge from '../'

it('renders correctly', () => {
    const component = <ESBadge count={3} />

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
