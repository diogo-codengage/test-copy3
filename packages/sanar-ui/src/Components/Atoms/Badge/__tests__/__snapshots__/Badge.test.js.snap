// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="es-badge es-badge__default es-badge__border ant-badge ant-badge-not-a-wrapper"
>
  <sup
    className="ant-scroll-number ant-badge-count"
    data-show={true}
    title={3}
  >
    <span
      className="ant-scroll-number-only"
      style={
        Object {
          "WebkitTransform": "translateY(-1300%)",
          "msTransform": "translateY(-1300%)",
          "transform": "translateY(-1300%)",
          "transition": "none",
        }
      }
    >
      <p
        className=""
      >
        0
      </p>
      <p
        className=""
      >
        1
      </p>
      <p
        className=""
      >
        2
      </p>
      <p
        className=""
      >
        3
      </p>
      <p
        className=""
      >
        4
      </p>
      <p
        className=""
      >
        5
      </p>
      <p
        className=""
      >
        6
      </p>
      <p
        className=""
      >
        7
      </p>
      <p
        className=""
      >
        8
      </p>
      <p
        className=""
      >
        9
      </p>
      <p
        className=""
      >
        0
      </p>
      <p
        className=""
      >
        1
      </p>
      <p
        className=""
      >
        2
      </p>
      <p
        className="current"
      >
        3
      </p>
      <p
        className=""
      >
        4
      </p>
      <p
        className=""
      >
        5
      </p>
      <p
        className=""
      >
        6
      </p>
      <p
        className=""
      >
        7
      </p>
      <p
        className=""
      >
        8
      </p>
      <p
        className=""
      >
        9
      </p>
      <p
        className=""
      >
        0
      </p>
      <p
        className=""
      >
        1
      </p>
      <p
        className=""
      >
        2
      </p>
      <p
        className=""
      >
        3
      </p>
      <p
        className=""
      >
        4
      </p>
      <p
        className=""
      >
        5
      </p>
      <p
        className=""
      >
        6
      </p>
      <p
        className=""
      >
        7
      </p>
      <p
        className=""
      >
        8
      </p>
      <p
        className=""
      >
        9
      </p>
    </span>
  </sup>
</span>
`;
